lib/game_client/ogs/ogs_game.dart

```diff
@@ -434,7 +443,7 @@ class OGSGame extends Game {
     _logger.fine('Received removed stones set for game $id');
 
     try {
-      _recentlyRemovedStonesString = data['stones'] as String? ?? '';
+      _recentlyRemovedStonesString = data['all_removed'] as String? ?? '';
```

Comment:
This is a semi-related bugfix: we need to store net (`all_removed`), not diff (`stones`).